tests/test_bitmaps.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+#coding:utf-8
+# Author:  Ross Golder --<ross@golder.org>
+# Purpose: test_bitmaps
+# Created: 23/04/2014
+# Copyright (C) 2014, Ross Golder
+# License: BSD licence

```

Comment:
I just copied this over from another test and modified it. I didn't add this. I really don't care if my name's not on the patch.